src/plugins/share/public/components/tabs/export/index.tsx

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { type IModalTabDeclaration } from '@kbn/shared-ux-tabbed-modal';
+import { ShareMenuItem } from '../../../types';
+import { ExportContent } from './export_content';
+import { useShareTabsContext } from '../../context';
+
+type IExportTab = IModalTabDeclaration;
+
+const exportTabReducer: IExportTab['reducer'] = (state, action) => {
+  switch (action.type) {
+    default:
+      return state;
+  }
+};
```

Comment:
I dont think we don't need this anymore since we aren't doing anything with state